cpu/cortexm_common/cortexm_init.c

```diff
@@ -26,6 +28,8 @@
  */
 #define FULL_FPU_ACCESS         (0x00f00000)
 
+extern const void *interrupt_vector[];
```

Comment:
Yes, I meant overriding the location specified in the default I assumed existed.